heka/meta/heka.yml

```diff
@@ -377,11 +377,17 @@ aggregator:
       module_file: /usr/share/lma_collector/filters/influxdb_accumulator.lua
       module_dir: /usr/share/lma_collector/common;/usr/share/heka/lua_modules
       preserve_data: false
-      message_matcher: "Type == 'heka.sandbox.gse_metric'"
+      message_matcher: "Type == 'heka.sandbox.gse_metric' || Type == 'heka.sandbox.metric'"
```

Comment:
Because the Type of messages injected by the annotation filter is "metric". See https://github.com/tcpcloud/salt-formula-heka/blob/stacklight/heka/files/lua/filters/influxdb_annotation.lua#L71. So without this change the InfluxDB accumulator and output will not see the annotation messages.